deploy/helm/README.md

```diff
@@ -195,43 +436,7 @@ otel:
         - attributes["k8s.object.kind"] == "ConfigMap" and resource.attributes["k8s.namespace.name"] != "kube-system"
 ```
 
-
-
-## Receive 3rd party metrics
-
-SWO K8s Collector has an OTEL service endpoint which is able to forward metrics and logs into SolarWinds Observability. All incoming data is properly associated with current cluster. Additionally, metrics are decorated with prefix `k8s.`.
-
-Service endpoint is provided in format
-
-```text
-"<chart-name>-metrics-collector.<namespace>.svc.cluster.local:4317"
-```
-
-### OpenTelemetry Collector configuration example
-
-In case you want to send data from your own [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector-contrib) into SWO you can either send them directly into [public OTLP endpoint](https://documentation.solarwinds.com/en/success_center/observability/content/configure/configure-otel-directly.htm) or you can send them via our `swo-k8s-collector` to have better binding to other data available in SolarWinds Observability. To do that add following exporter into your configuration.
-
-```yaml
-config:
-  exporters:
-    otlp:
-      endpoint: <chart-name>-metrics-collector.<namespace>.svc.cluster.local:4317
-```
-
-### Telegraf configuration example
-
-[Telegraf](https://github.com/influxdata/telegraf) is a plugin-driven server agent used for collecting and reporting metrics.
-
-Telegraf metrics can be sent into our endpoint by adding following fragment to your `values.yaml`
-
-```yaml
-config:
-  outputs:
-    - opentelemetry:
-        service_address: <chart-name>-metrics-collector.<namespace>.svc.cluster.local:4317
-```
-
-## Auto Instrumentation (experimental feature)
```

Comment:
This section is referenced in SWO website. We will need to update the links there, too.

Reply:
There is separate page for it now: https://documentation.solarwinds.com/en/success_center/observability/content/intro/kubernetes/auto-instrumentation.htm

Reply:
I created PR: https://github.com/solarwinds-cloud/nighthawk-clients/pull/16810